dice/apps/rounds/models.py

```diff
@@ -28,15 +23,15 @@ class Round(models.Model):
     game = models.ForeignKey('games.Game', on_delete=models.CASCADE)
     user = models.ForeignKey('users.User', on_delete=models.CASCADE)
     dice1 = models.OneToOneField(Dice, on_delete=models.CASCADE, related_name='round_as_first_dice',
-                                 default=Dice.create)
+                                 default=Dice.objects.create)
```

Comment:
to make sure django properly serialize `default` function let's use refular function stored somewhere in separate module (e.g. in `apps.<app_name>.logic.migrations.defaults`). This function should be as simple as possible, in your case:

```python
def default_dice_factory():
    """Factory default value for `dice*` fields in ``Round`` model."""
    return Dice.objects.create()
``` 

In the future we will talk about other better approaches to implement dices in round